pkg/config/config.go

```diff
@@ -1108,6 +1108,10 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("container_image.enabled", false)
 	bindEnvAndSetLogsConfigKeys(config, "container_image.")
 
+	// Remote process collector
+	config.BindEnvAndSetDefault("workloadmeta.remote_process_collector.enabled", false)
+	bindEnvAndSetLogsConfigKeys(config, "workloadmeta.")
```

Comment:
That's not necessary, we're not sending to logs intake.

Reply:
It should be fixed in the last commits